updatecli/updatecli.d/karpenter-helm-release.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: Bump `karpenter` helm release version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastChartVersion:
+    name: Karpenter AWS Provider Helm Chart Latest Version
+    kind: githubrelease
+    spec:
+      owner: aws
+      repository: karpenter-provider-aws
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: latest
+    transformers:
+      - trimprefix: "v"
```

Comment:
Same as https://github.com/jenkins-infra/terraform-aws-sponsorship/pull/104/files/def043a6c343e7479b42a3431d31186eb2014a9b#r1923504399 (you want the source to track an Helm chart, not a github release)